packages/adblocker/src/filters/network.ts

```diff
@@ -966,6 +974,30 @@ export default class NetworkFilter implements IFilter {
           }
         }
       }
+
+      if (domainsList.size !== 0) {
+        domains = Domains.parse(Array.from(domainsList).join('|'), {
```

Comment:
Would it make sense to change the API of `Domains.parse` to avoid having to re-join them but instead pass a `Set` or array?

Reply:
updated in 043710ab7a36208b4818e91b32ee8d313c225aba